tests/blockchain/test_blockchain_transactions.py

```diff
@@ -14,7 +14,7 @@
 from chia.util.ints import uint64
 from tests.blockchain.blockchain_test_utils import _validate_and_add_block
 from tests.wallet_tools import WalletTool
-from tests.setup_nodes import bt, setup_two_nodes, test_constants
```

Comment:
There are a number of warts & limitations in the Python fixtures implementation -
Removing these explicit references and putting them in `conftest.py` is the accepted way of using fixtures.

Reply:
currently `bt` is a global variable that's being imported here. in the new world, `bt` is a fixture, so I don't think you *can* import it (you certainly don't want to)